src/server/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+IF(WIN32)
```

Comment:
I would not put the whole bunch of sources in if-else clause. Can we define extra variable for WIN32?

```
if(WIN32)
set(WIN32_SOURCES coutbuf.cpp)
endif(WIN32)

target(... WIN32_SOURCES)
```

I believe in this case cmake will add extra sources to the target in case WIN32 otherwise it will be just empty value.

Please note also that we use lower case for cmake commands i.e. _set_, _if_ etc